reconcile.go

```diff
@@ -9,6 +9,10 @@ import (
 
 // reconcile calculates and then applies promotions and demotions
 func (a *Autopilot) reconcile() error {
+	if !a.ReconciliationEnabled() {
```

Comment:
The main autopilot routine still runs and even keeps up with the interval on which it is supposed to reconcile things. We prevent that reconciliation/raft modifications from happening by checking a flag to see if its enabled.